app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +324,160 @@ export default function () {
           </Button>
         </Form>
         {featureCircularsFilterByDate && (
-          <DateSelectorButton startDate={startDate} endDate={endDate} />
+          <>
+            <DateSelectorButton
+              startDate={startDate}
+              endDate={endDate}
+              onClick={() => { setShowContent((shown) => !shown) }}
+              expanded={showContent}
+            />
+            {showContent && (
+              <DetailsDropdownContent className="margin-top-1">
+                <div className="usa-radio maxw-card-xlg">
+                  <div className="display-flex flex-row">
+                    <div className="display-flex flex-column flex-align-start margin-1">
+                      <Radio
+                        id="radio-alltime"
+                        name="radio-date"
+                        value="undefined"
+                        defaultChecked={true}
+                        label="All Time"
+                        className="usa-search__filter-button"
```

Comment:
Remove for all radio buttons. This is not a [USWDS Search component](https://designsystem.digital.gov/components/search/).

```suggestion
```

Reply:
ah, must've been vestigial from when I was trying to use buttons in the dropdown